src/UniversalDashboard/Cmdlets/Inputs/NewInputFieldCommand.cs

```diff
@@ -27,6 +27,9 @@ public class NewInputFieldCommand : PSCmdlet
         [Parameter()]
         public string[] Placeholder { get; set; }
 
+        [Parameter()]
+        public SwitchParameter Disabled { get; set; } = false;
```

Comment:
You don't need to set this to false. I also don't know what setting to false will do. If it's working, probably nothing.

Reply:
That was a left over thing from me having it a bool and not a switch